src/server/AppManager.ts

```diff
@@ -414,7 +414,7 @@ export class AppManager {
     }
 
     public async add(appPackage: Buffer, installationParameters: IAppInstallParameters): Promise<AppFabricationFulfillment> {
-        const { enable, marketplaceInfo, permissionsGranted } = installationParameters;
+        const { enable = true, marketplaceInfo, permissionsGranted } = installationParameters;
```

Comment:
Fixes the app getting disabled after installation. Having this `enable` with no default param, the app only gets initialized and not enabled.